Sources/Utilities/ImageTools/ImageTools.swift

```diff
@@ -0,0 +1,358 @@
+//
+//  ImageTools.swift
+//  PovioKit
+//
+//  Created by Borut Tomazin on 13/06/2024.
+//  Copyright © 2024 Povio Inc. All rights reserved.
+//
+
+import UIKit
+import PovioKitCore
+
+/// A utility class for performing various image-related operations.
+public final class ImageTools: NSObject {
+#if canImport(Kingfisher)
+  private var prefetcher: ImagePrefetcher?
+#endif
+  
+  override public init() {
+    super.init()
+  }
+}
```

Comment:
Do we need this or can we just extend `UIImage`? 🤔

Reply:
We need it, it's used in the `prefetch` method.

Reply:
- saveToPhotoLibrary
- downsize
- compress
- ...

could just be UIImage extensions, or am I missing something?

Reply:
Good point. It makes sense to have UIImage extension instead. I've separated into two. `UIImage+PovioKit` where all UIKit operations are happening and `UIImage+Kingfisher` for library specific operations.